src/redux/actions/articleActions.js

```diff
@@ -42,6 +52,18 @@ export const getArticleBySlug = slug => async (dispatch) => {
   }
 };
 
+export const createArticleComment = (slug, comment) => async (dispatch) => {
+  dispatch(articleLoading());
+  try {
+    const token = localStorage.getItem('jwtToken');
+    SetAuthToken(`Bearer ${token}`);
```

Comment:
Why is this here?

Reply:
I would refactor it, I was trying to make it work